app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "com.example.atheneum"
-        minSdkVersion 18
+        minSdkVersion 21

```

Comment:
I don't have a problem with this but I'm curious why you needed to change the API level.